udata_front/theme/gouvfr/assets/js/pages/OrganizationPublishingForm/Step2DescribeOrganization.vue

```diff
@@ -0,0 +1,348 @@
+  }>();
+
+  const { id: nameOrganizationAccordionId } = useUid("accordion");
+  const { id: addAcronymAccordionId } = useUid("accordion");
+  const { id: addDescriptionAccordionId } = useUid("accordion");
+  const { id: addSiretAccordionId } = useUid("accordion");
+  const { id: addWebsiteAccordionId } = useUid("accordion");
+  const { id: addLogoAccordionId } = useUid("accordion");
+
+  const organization = reactive({...props.organization});
+  const file = ref(null);
+
+  const checkOrga = ref({
+    name: '',
+    siren: '',
+    isPublicService: '',
+    exists: null as boolean | null
+  });
+
+  const checkBusinessId = (value) => {
+    return !value || checkOrga.value.exists;
+  };
+
+  const requiredRules = {
+    business_number_id: { checkBusinessId },
+    description: { required },
+    name: { required },
+    url: { url },
+  };
+
+  const warningRules = {
+    acronym: {},
+    business_number_id: { checkBusinessId },
+    description: {required, minLengthValue: minLengthWarning(quality_description_length) },
+    logo: {},
+    name: { required },
+    url: { url },
+  };
+
+  const { getErrorText, getFunctionalState, getWarningText, hasError, hasWarning, validateRequiredRules, v$, vWarning$ } = useFunctionalState(organization, requiredRules, warningRules);
+
+  /**
+   * @type {import("vue").ComputedRef<Record<string, import("../../types").PublishingFormAccordionState>>}
+   */
+  const state = computed(() => {
+    return {
+      acronym: vWarning$.value.acronym.$dirty ? "info" : "disabled",
+      business_number_id: getFunctionalState(vWarning$.value.business_number_id.$dirty, v$.value.business_number_id.$invalid, vWarning$.value.business_number_id.$error),
+      description: getFunctionalState(vWarning$.value.description.$dirty, v$.value.description.$invalid, vWarning$.value.description.$error),
+      logo: vWarning$.value.logo.$dirty ? "info" : "disabled",
+      name: getFunctionalState(vWarning$.value.name.$dirty, v$.value.name.$invalid, vWarning$.value.name.$error),
+      url: getFunctionalState(vWarning$.value.url.$dirty, v$.value.url.$invalid, vWarning$.value.url.$error),
+    };
+  });
+
+  /**
+   *
+   * @param {string} field
+   */
+  const fieldHasError = (field) => hasError(state, field);
+
+  /**
+   *
+   * @param {string} field
+   */
+  const fieldHasWarning = (field) => hasWarning(state, field);
+
+  const submit = () => {
+    validateRequiredRules().then(valid => {
+      if(valid) {
+        emit("next", organization, file);
+      }
+    });
+  };
+
+  /**
+   *
+   * @param {Array<import("../../types").NewDatasetFile>} newFiles
+   */
+    const addFiles = (newFile) => {
+      file.value = newFile;
+    };
+
+  watch(() => organization.business_number_id, (newValue) => {
+    if (newValue.length === 14) {
+      axios.get("https://recherche-entreprises.api.gouv.fr/search", {
+        params: {
+          q: newValue
+        }
+      })
```

Comment:
Maybe remove spaces here if someone copy a SIRET with spaces as it's often the case